engines/advancedDetector.cpp

```diff
@@ -556,6 +556,15 @@ static bool getFilePropertiesIntern(uint md5Bytes, const AdvancedMetaEngine::Fil
 
 		if (fileProps.size != 0)
 			return true;
+
+		Common::SeekableReadStream *dataFork = macResMan.getDataFork();
```

Comment:
So, this now effectively computes md5 for the _DATA_ fork when resource fork size is 0 and `ADGF_MACRESFORK` flag is set. This does not sound right.

In fact, in this case, the file should not be in MacBinary format at all, because DataFork-only is basically a plain file.

This makes this approach illogical and not correct.